### PR TITLE
Application Form

### PR DESCRIPTION
**Pull request by AzizRAA**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+legend {
+  font-weight: 700;
+}
+
+#number {
+  width: 150px;
+}
+
+
```
Comment (JOdiemo) [inline style.css:43]: There are double spaces. The convention is to have a single space. Kindly look onto this

**Review by JOdiemo — CHANGES_REQUESTED** [review]
Good job so far!
But there are some required changes below:

1. Please read my comments and make the changes.
2. Please check for linter errors before uploading your project
mentioned.
3. Mention comments on each commit.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,81 @@
+body {
+  font-family: 'Anaheim';
+  font-size: 1.2em;
```
Comment (Dev-inspired) [inline style.css:3]: could you kindly use different  font-family to make your application look professional.

**Review by Dev-inspired — COMMENTED** [review]
Good documentation.

**Pull request merged**